apple/testing/default_runner/macos_test_runner.template.sh

```diff
@@ -160,13 +166,41 @@ xcrun llvm-cov \
   "$test_binary" \
   @"$COVERAGE_MANIFEST" \
   > "$COVERAGE_OUTPUT_FILE" \
-  2> "$error_file" \
-  || llvm_cov_status=$?
+  2> "$export_error_file" \
+  || llvm_cov_export_status=$?
+
+llvm_cov_json_export_status=0
+readonly json_export_error_file="$TEST_TMP_DIR/llvm-cov-json-export-error.txt"
+if [ "$COVERAGE_PRODUCE_JSON" = true ]; then
+  if [[ -n "${TEST_UNDECLARED_OUTPUTS_DIR}" ]]; then
```

Comment:
Can this ever not be set?

Reply:
This was copied from one of the other runner shell scripts, down tor remove it